src/theme.js

```diff
@@ -1,6 +1,77 @@
-
+import { Link } from "gatsby"
+import React from "react"
 import styled from 'styled-components'
 
+import "./fonts/Inter/inter.css"
```

Comment:
we could remove fonts from source and add them as npm dependency instead
https://www.npmjs.com/package/typeface-inter

maybe even better, from gatsby google fonts plugin
https://www.gatsbyjs.com/plugins/gatsby-plugin-google-fonts/